geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -9992,7 +10000,48 @@ public EntryExpiryTask getEntryExpiryTask(Object key) {
     return br.getEntryExpiryTask(key);
   }
 
+  void updatePRNodeInformation() {
+    updatePartitionRegionConfig(prConfig -> {
+      CacheLoader cacheLoader = basicGetLoader();
+      CacheWriter cacheWriter = basicGetWriter();
+      if (prConfig != null) {
+
+        for (Node node : prConfig.getNodes()) {
+
+          if (node.getMemberId().equals(getMyId())) {
+            node.setLoaderAndWriter(cacheLoader, cacheWriter);
+          }
+        }
+      }
+      return prConfig;
+    });
+  }
+
   public Logger getLogger() {
     return logger;
   }
+
+  interface PartitionRegionConfigModifier {
+    PartitionRegionConfig modify(PartitionRegionConfig prConfig);
```

Comment:
This can be `void modify(PartitionRegionConfig prConfig);`. Because we longer use the return value.